src/DynamicData/Cache/ObservableCacheEx.VirtualiseAndPage.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Copyright (c) 2011-2023 Roland Pheasant. All rights reserved.
+// Roland Pheasant licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using System.Reactive.Linq;
+using DynamicData.Cache.Internal;
+
+namespace DynamicData;
+
+/// <summary>
+/// ObservableCache extensions for the virtualised group of operators.
+/// </summary>
+public static partial class ObservableCacheEx
+{
+    /// <summary>
+    /// Sort and virtualize the underlying data from the specified source.
+    /// </summary>
+    /// <typeparam name="TObject">The type of the object.</typeparam>
+    /// <typeparam name="TKey">The type of the key.</typeparam>
+    /// <param name="source">The source.</param>
+    /// <param name="comparer">The comparer to order the resulting dataset.</param>
+    /// <param name="virtualRequests">The virtualizing requests.</param>
+    /// <returns>An observable which will emit virtual change sets.</returns>
+    /// <exception cref="ArgumentNullException">source.</exception>
+    public static IObservable<IChangeSet<TObject, TKey, VirtualContext<TObject>>> SortAndVirtualize<TObject, TKey>(this IObservable<IChangeSet<TObject, TKey>> source,
```

Comment:
I am torn on this one. Adding the SortAnd... clearly marks it apart from the old way. However, you're absolutely correct that it may not be required at all. That also holds true for the SortAndBind operator. Do we just rename these back to the original versions without the SortAnd prefix?  I think we probably need a face to face on this question

Reply:
Using just `.Virtualize()` and `.Bind()` only works if there's no ambiguity at the call site.

```cs
using var subscription = someStream
    .SortAndBind(out var target)
    .Subscribe();
```

Right now, this is a valid invocation of the operator, if the items in `someStream` are `IComparable<TObject>`. If we were to swap `.SortAndBind()` to just `.Bind()` we lose a fair bit of clarity, I think, that there's a sort happening. And what if the user wants to opt-out of that implicit sort? Is that even possible?

I'm not sure what the equivalent scenario would be for virtualization, but that's the perspective we want to look at, to consider `.SortAndVirtualize()` versus just `.Virtualize()`.